src/collector.rs

```diff
@@ -58,20 +55,84 @@ impl Collector {
 
         self.hptr.set(ptr);
         self.hstart.set(ptr as usize);
-        self.hend.set(ptr as usize + HSIZE);
+        self.hend.set(ptr as usize + size);
     }
 
     pub fn mk_root_table<P: AsRef<Path>>(&self, path: P) {
         unimplemented!()
     }
 
-    // Perform the actual garbage collection. We use the name `reclaim` to
-    // disambiguate from Rust's notion of `collect` on iterators.
+    /// Perform the actual garbage collection. We use the name `reclaim` instead
+    /// of collect to disambiguate from Rust's notion of `collect` on iterators.
     pub(crate) fn reclaim(&self) {
-        unimplemented!()
+        eprintln!("Collection is no-op: not yet implemented")
```

Comment:
`reclaim is a no-op`?

Reply:
I mean this in the sense that at the moment, collection does nothing (But `unimplemented!` will panic, which is not desirable)